docs/howto/custom-template-tags.txt

```diff
@@ -499,6 +499,8 @@ positional arguments. For example:
 
     {% my_tag 123 "abcd" book.title warning=message|lower profile=user.profile %}
 
+.. _howto-custom-template-tags-simple-tag-assignment:
```

Comment:
> I think it's confusing to put links to places in the docs that don't have a heading.
Understood. I could link to the simple tags heading, but it's quite a long section. Do you think that's ok, or is it possible to create a subheading?

> We can remove the ".. _howto-custom-template-tags-assignment-tags:" too and the remaining usage in the 1.4 release notes.
OK, I will do this on Thursday.